ansible/configs/satellite-vm/default_vars.yml

```diff
@@ -0,0 +1,61 @@
+---
+
+env_type: satellite-vm 
+output_dir: /tmp/workdir                # Writable working scratch directory
+email: "{{env_type}}@example.com"
+guid: defaultguid
+
+# for file based repo
+repo_method: file 
+use_own_repos: true
+repo_version: "6.4"
```

Comment:
Why such an old repo? Any reason not to update so you get updated packages with security and other fixes?

Reply:
Yeah, This is repo_version=6.4 is for satellite v6.4 and as of now we are doing only satellite 6.4. Hence no need to update, we need to keep it as is.